examples/getting_started/5-Live_Data.ipynb

```diff
@@ -207,10 +217,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%opts Curve (line_width=0.03 color='blue') Points (color='red' size=10) Curve.Init (color='red' line_width=2)\n",
+    "from holoviews.streams import PointerXY\n",
+    "\n",
     "# When run live, this cell's output should match the behavior of the GIF below\n",
-    "dmap = hv.DynamicMap(interactive_clifford, kdims=['a','b','c','d'], streams=[PointerXY(x=0,y=0)])\n",
-    "dmap.redim.range(a=(-1.4,-1),b=(1.6,1.8),c=(1,1.5),d=(0.7,0.8))"
+    "iclifford = hv.DynamicMap(interactive_clifford, kdims=['a','b','c','d'], streams=[PointerXY(x=0,y=0)])\n",
+    "iclifford.redim.range(a=(-1.4,-1),b=(1.6,1.8),c=(1,1.5),d=(0.7,0.8), x=(-2,2), y=(-2,2))"
```

Comment:
What's up with x padding here? I think mixing redim for the DynamicMap dimensions and to apply padding is confusing. I'd prefer using the padding plot option instead.

Reply:
To me it seems simpler to sneak in the x,y padding here rather than draw attention to it using the padding plot option.  It's not really padding, anyway, at least not just for visual appearance; it's to account for the fact that some values of the parameters end up with plots larger than others, and the initial one doesn't cover the full range expected across all parameters.

Reply:
I'd use ``xlim`` and ``ylim`` in that case but I'm okay with sneaking it in.